### PR TITLE
Cask/bananas v0.0.12

### PR DESCRIPTION
**Pull request by bobbymannino**
I think we got there in the end, styled properly, the developer signed the app

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/b/bananas.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask "bananas" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/mistweaverco/bananas/releases/latest/download/bananas_universal.dmg",
```
Comment (bevanjkay) [inline Casks/b/bananas.rb:5]:
```suggestion
  url "https://github.com/mistweaverco/bananas/releases/download/v#{version}/bananas_universal.dmg"

File: Casks/b/bananas.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+cask "bananas" do
+  version :latest
```
Comment (bevanjkay) [inline Casks/b/bananas.rb:2]:
```suggestion
  version "0.0.12"

File: Casks/b/bananas.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  sha256 :no_check
```
Comment (bevanjkay) [inline Casks/b/bananas.rb:3]:
```suggestion
  sha256 "aeb3d6ab40937524a28a0a569a4d03247d256255641c959dfa97f2c19e725ed8"

File: Casks/b/bananas.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+  url "https://github.com/mistweaverco/bananas/releases/latest/download/bananas_universal.dmg",
+      verified: "github.com/mistweaverco/bananas/"
+  name "bananas"
+  desc "Cross-Platform screen sharing made simple"
```
Comment (bevanjkay) [inline Casks/b/bananas.rb:8]:
```suggestion
  desc "Cross-platform screen sharing tool"

File: Casks/b/bananas.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+cask "bananas" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/mistweaverco/bananas/releases/latest/download/bananas_universal.dmg",
+      verified: "github.com/mistweaverco/bananas/"
+  name "bananas"
```
Comment (bevanjkay) [inline Casks/b/bananas.rb:7]:
```suggestion
  name "Bananas Screen Sharing"

**Review by bobbymannino — COMMENTED** [review]
These were @bevanjkay's changes;

- changing from latest to v0.0.12
- added checksum
- neatened verbiage up to match brew styles

**Comment by bobbymannino** [comment]
im just making a mess of things over here, if possible may someone add this cask to the repo, every step i take i encounter a new issue that makes everything more complicated, thank you

**Pull request closed without merging**